tests/raspbian/test_interfaces.py

```diff
@@ -0,0 +1,471 @@
+import unittest
+
+from netjsonconfig import Raspbian
+from netjsonconfig.utils import _TabsMixin
+
+
+class TestInterfacesRenderer(unittest.TestCase, _TabsMixin):
+
+    def test_ipv4_static(self):
+        o = Raspbian({
+            "interfaces": [
+                {
+                    "name": "eth0",
+                    "type": "ethernet",
+                    "addresses": [
+                        {
+                            "family": "ipv4",
+                            "proto": "static",
+                            "address": "10.0.0.1",
+                            "mask": 28
+                        }
+                    ]
+                }
+            ]
+        })
+
+        expected = '''# config: /etc/network/interfaces
+
+auto eth0
+iface eth0 inet static
```

Comment:
why in the examples contained in the docs we have indentation for these? Ensure the documentation returns the same output the library generates please, otherwise users will get confused